pkg/lib/authenticationflow/declarative/input_step_identify.go

```diff
@@ -96,6 +99,8 @@ func (i *InputSchemaStepIdentify) MakeInput(rawMessage json.RawMessage) (authflo
 type InputStepIdentify struct {
 	Identification config.AuthenticationFlowIdentification `json:"identification,omitempty"`
 
+	IDToken string `json:"id_token,omitempty"`
```

Comment:
It seems in reauth, the id token is from id_token_hint of the oauth session. I am not sure if it could be difficult for the ui to pass in id token as an input.

Reply:
Oh I saw id_token_hint was forwarded to ui in this pr.